netket/machine/jax.py

```diff
@@ -324,6 +324,137 @@ def JaxRbm(hilbert, alpha, dtype=complex):
     )
 
 
+def JaxMpsPeriodic(
```

Comment:
I think that for consistency with the previous version (and also because we won't write a numpy version of this) it would be better to just call this ```MpsPeriodic``` without Jax!